### PR TITLE
Add runningaverage sample

### PR DESCRIPTION
**Pull request by pivotal-sukhil-suresh**

### CODE REVIEW

**Comment by trisberg** [comment]
LGTM, worked on my machine

File: samples/runningaverage/.gitignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+*
+!.gitignore
+!*.go
```
Comment (ericbottard) [inline samples/runningaverage/.gitignore]: All but `*.go` should be ignored. All other exclusions should not be present

File: samples/runningaverage/README.adoc [hunk 2]
```diff
@@ -0,0 +1,83 @@
+= runningaverage
+
+[horizontal]
+*Invoker*:: go
+*Function*:: link:runningaverage.go[runningaverage.go]
+*Input*:: numbers
+
+=== prerequisites
```
Comment (ericbottard) [inline samples/runningaverage/README.adoc]: section titles should be capitalized

File: samples/runningaverage/.gitignore (hunk 1 above)
Comment (ericbottard) [reply samples/runningaverage/.gitignore]: Sorry, thought this was `.dockerignore` (which is needed)

**Pull request merged**